contracts/periphery/contracts/s2s/calls/share/SystemCalls.sol

```diff
@@ -29,7 +29,7 @@ library SystemCalls {
             hex"0009",
             rmk
         );
-        uint256 weight = rmk.length.mul(1_000);
+        uint256 weight = rmk.length.mul(2_000);
```

Comment:
Please add some note about where to get this value.

Reply:
This calculation way comes from the weight declaration of the dispatch call on the chain. Note that the chain must use the correct version, or, the param may be incorrect. 
https://github.com/darwinia-network/substrate/blob/1df7fe589a94caa33e8b9290dd474613bc170416/frame/system/src/weights.rs#L68

Reply:
Hmm, I recommend adding some comments above the code instead of in this pr. :smile:

Reply:
ye